google/cloud/forseti/services/inventory/base/resources.py

```diff
@@ -1349,6 +1349,39 @@ def zone(self):
                          self._data)
             return None
 
+# Kubernetes resource class
+
+
+class KubernetesNode(resource_class_factory('kubernetes_node', 'id')):
```

Comment:
There is no 'id' attribute in the data from CAI. For all the kubernetes resources you need to use the hash of the "uid" under the "metadata" key. The current resource_class_factory doesn't support using a key inside of another key currently, so I recommend creating a special version of the resource_class_factory called k8s_resource_class_factory and have it handle generating the id from the uid.

Reply:
Done